cpp/modmesh/toggle/pymod/wrap_Toggle.cpp

```diff
@@ -54,19 +54,45 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapToggle
 
     WrapToggle(pybind11::module & mod, char const * pyname, char const * pydoc);
 
+    static std::string report(wrapped_type const & self);
+
 }; /* end class WrapToggle */
 
 WrapToggle::WrapToggle(pybind11::module & mod, char const * pyname, char const * pydoc)
     : base_type(mod, pyname, pydoc)
 {
     namespace py = pybind11;
+
+    (*this)
+        .def("report", &report)
+        //
+        ;
+
+    // Static properties.
     (*this)
         .def_property_readonly_static(
             "instance",
-            [](py::object const &) -> auto & {
-                return wrapped_type::instance();
-            })
-        .def_property("show_axis", &wrapped_type::get_show_axis, &wrapped_type::set_show_axis);
+            [](py::object const &) -> auto & { return wrapped_type::instance(); })
+        .def_property_readonly_static(
+            "USE_PYSIDE",
+            [](py::handle const &)
+            { return bool(wrapped_type::USE_PYSIDE); })
+        //
+        ;
+
+    // Instance properties.
+    (*this)
+        .def_property("show_axis", &wrapped_type::get_show_axis, &wrapped_type::set_show_axis)
+        //
+        ;
+}
+
+std::string WrapToggle::report(WrapToggle::wrapped_type const &)
```

Comment:
Add a simple status reporting helper.  It may grow to be very long in the future.  We will need a new design then, but it's OK to keep it simple for now.